cypress/integration/sort-by-dropdown.spec.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+import { getByCypressTag } from '../utils'
+
+describe('Sort By dropdown', function () {
+  beforeEach(() => {
+    cy.visit('http://localhost:8080/?dashboard=true')
+  })
+
+  it ('Is visible', function () {
+    getByCypressTag('sortDropdown').should('be.visible')
```

Comment:
The previous test was checking that the `Dashboard` was rendering a sorting dropdown, which I think this tests just fine. I briefly tried making an integration test for `DashboardApp` to see if it rendered the sorting dropdown, but ran into issues mounting a connected component.

Reply:
Also I used to get an email whenever a build failed, but I didn't get any today? I'll have to check the configuration.